templates/go/events/manifest.yaml

```diff
@@ -0,0 +1,12 @@
+builders:
+  default: paketobuildpacks/builder:base
```

Comment:
Note how all of the templates use the same builder.

Reply:
That's a nice simplification!